src/styles/blocks/antisnake.scss

```
@@ -0,0 +1,28 @@
+.antisnake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($block-height, 1fr));
+
+  @media (min-width:1200px) {
```

```suggestion
  @media (min-width:900px) {
```
Чому 1200? Ширина одного блока - 300px, отже тобі треба відштовхуватись від 900px